src/pages/UnlinkLoginPage.tsx

```diff
@@ -0,0 +1,48 @@
+import {StackScreenProps} from '@react-navigation/stack';
+import React, {useEffect} from 'react';
+import {type OnyxEntry, withOnyx} from 'react-native-onyx';
+import FullScreenLoadingIndicator from '@components/FullscreenLoadingIndicator';
+import usePrevious from '@hooks/usePrevious';
+import Navigation from '@libs/Navigation/Navigation';
+import {PublicScreensParamList} from '@navigation/types';
+import * as Session from '@userActions/Session';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import SCREENS from '@src/SCREENS';
+import type {Account} from '@src/types/onyx';
+
+type UnlinkLoginPageOnyxProps = {
+    /** The details about the account that the user is signing in with */
+    account: OnyxEntry<Account>;
+};
+
+type UnlinkLoginPageProps = UnlinkLoginPageOnyxProps & StackScreenProps<PublicScreensParamList, typeof SCREENS.UNLINK_LOGIN>;
+
+function UnlinkLoginPage({route, account}: UnlinkLoginPageProps) {
+    const accountID = route.params.accountID ?? '';
```

Comment:
should't we cast it to number as in the codebase `accountID` is always a number?

Reply:
In this case, `accountID` is a string since it was passed through the route params

Reply:
According to our codebase `accountID` is always a number 

I think we need to cast it here to have consistency and leave unlinkLogin with number only

Reply:
`unlinkLogin` function is used only in one place, and it uses `accountID` as a string in the current app version. At the same time I agree that it's better to keep it consistent, I'm going to test it a little with string/number values